ansible/playbooks/sap-hana-system-replication-hooks.yaml

```diff
@@ -53,7 +64,7 @@
       block:
         - name: Ensure ha_dr_provider_SAPHanaSR section exists in global.ini
           community.general.ini_file:
-            path: /usr/sap/HDB/SYS/global/hdb/custom/config/global.ini
+            path: /usr/sap/{{ sap_hana_install_sid }}/SYS/global/{{ sap_hana_install_sid | lower }}/custom/config/global.ini
```

Comment:
This needs to be fixed, because path does not contain {{ sap_hana_install_sid | lower }} , instead it is fixed "hdb"

Example of my H01 hana:
/usr/sap/H01/SYS/global/hdb/custom/config/global.ini